src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -83,6 +81,13 @@ void emberAfThermostatClusterServerInitCallback()
     // with weak binding so that real thermostat
     // can get the values.
     // or should this just be the responsibility of the thermostat application?
+
+    // Feature map override, for testing only:
+    uint32_t featureMap = FEATURE_MAP_HEAT | FEATURE_MAP_COOL | FEATURE_MAP_AUTO;
```

Comment:
Why are we overriding the value set in the actual app configuration (in ZAP) by default?  That does not seem right...  I know the current code is doing it, but a PR just landed to stop that, yes?

Reply:
This remains only for testing purposes.  Rebase and regen process to test code changes that are dependent on Feature Map contents are very time consuming.